### PR TITLE
OZ-3916 Update api docs for update album API

### PR DESCRIPTION
**Pull request by shagupt-adobe**

### CODE REVIEW

**Comment by shagupt-adobe** [comment]
@slohia-adobe Pl review.

File: docs/api/LightroomPartnerAPIsSpec.json [hunk 1]
```diff
@@ -1110,6 +1110,116 @@
             "$ref":"#/components/responses/ServiceUnavailable"
           }
         }
+      },
+      "post": {
+        "tags": [
+          "albums"
+        ],
+        "summary": "Update album",
+        "description": "Update an existing album.",
```
Comment (dkatyal) [inline docs/api/LightroomPartnerAPIsSpec.json]: Good to add this info: the existing album should be self created (serviceId check) and of project and project set subtypes.

File: docs/api/LightroomPartnerAPIsSpec.json [hunk 2]
```diff
@@ -1110,6 +1110,116 @@
...
+            "description": "Identifier for the album.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "Album metadata and information to be updated.",
+            "required": true,
+            "schema": {
```
Comment (dkatyal) [inline docs/api/LightroomPartnerAPIsSpec.json:1158]: only payload should be mentioned.

File: docs/api/LightroomPartnerAPIsSpec.json [hunk 3]
```diff
@@ -1110,6 +1110,116 @@
...
+              "properties": {
+                "subtype": {
+                  "type": "string",
+                  "enum": [ "project", "project_set" ]
+                },
+                "serviceId": {
+                  "type": "string",
+                  "description": "The API Key (client identifier) of the service creating the album."
+                },
+                "payload": {
```
Comment (dkatyal) [inline docs/api/LightroomPartnerAPIsSpec.json:1161]: changes in payload should not be limited. anything inside payload can be updated.

**Review by slohia-adobe — APPROVED** [review]

**Review by dkatyal — APPROVED** [review]

**Review by slohia-adobe — APPROVED** [review]

**Pull request merged**